app/experimenter/experiments/tests/api/v5/utils.py

```diff
@@ -0,0 +1,11 @@
+from collections import OrderedDict
+
+
+def object_to_ordered_dict(obj, keys):
+    return OrderedDict({k: getattr(obj, k) for k in keys})
+
+
+def branch_to_dict(branch):
+    return object_to_ordered_dict(
+        branch, ["name", "description", "ratio", "feature_enabled", "feature_value"]
+    )
```

Comment:
Yeah I see your point but I generally try to make assertions specifically unordered, so we could do the other way around, convert OrderedDict to Dict, and for lists, use membership assertions.